dune

```diff
@@ -27,380 +28,327 @@
-   lazy_backtrack diffing diffing_with_keys
-=======
-   profile terminfo ccomp warnings consistbl strongly_connected_components
-   targetint load_path int_replace_polymorphic_compare binutils local_store
-   lazy_backtrack diffing diffing_with_keys unit_info
->>>>>>> 5.2.0
-
-   ;; PARSING
-<<<<<<< HEAD
-   location longident docstrings printast syntaxerr ast_helper
-   camlinternalMenhirLib ast_iterator parser lexer parse pprintast ast_mapper
-   attr_helper builtin_attributes ast_invariants depend jane_syntax_parsing
-   jane_syntax ; manual update: mli only files asttypes parsetree
-||||||| 121bedcfd2
-   location longident docstrings syntaxerr ast_helper camlinternalMenhirLib
-   parser lexer parse printast pprintast ast_mapper ast_iterator attr_helper
-   builtin_attributes ast_invariants depend
-   ; manual update: mli only files
-   asttypes parsetree
-=======
-   location longident docstrings syntaxerr ast_helper camlinternalMenhirLib
-   ast_iterator builtin_attributes parser lexer parse printast pprintast
-   ast_mapper attr_helper ast_invariants depend
-   ; manual update: mli only files
-   asttypes parsetree
->>>>>>> 5.2.0
-
-   ;; TYPING
-   ident path jkind primitive shape shape_reduce types btype oprint subst predef datarepr
-   global_module cmi_format persistent_env env errortrace mode jkind_types jkind_intf
-   typedtree printtyped ctype printtyp includeclass mtype envaux includecore
-   tast_iterator tast_mapper signature_group cmt_format cms_format untypeast
-   includemod includemod_errorprinter
-<<<<<<< HEAD
-   typemode typetexp patterns printpat parmatch stypes typedecl typeopt
-   value_rec_check typecore solver_intf solver mode_intf mode uniqueness_analysis
-   typeclass typemod typedecl_variance typedecl_properties
-   typedecl_separability cmt2annot
-||||||| 121bedcfd2
-   typetexp patterns printpat parmatch stypes typedecl typeopt rec_check
-   typecore
-   typeclass typemod typedecl_variance typedecl_properties typedecl_immediacy
-   typedecl_unboxed typedecl_separability cmt2annot
-=======
-   typetexp patterns printpat parmatch stypes typeopt typedecl value_rec_check
-   typecore
-   typeclass typemod typedecl_variance typedecl_properties typedecl_immediacy
-   typedecl_unboxed typedecl_separability cmt2annot
->>>>>>> 5.2.0
-   ; manual update: mli only files
-   annot outcometree value_rec_types
-
-   ;; lambda/
-   debuginfo lambda matching printlambda runtimedef runtimetags tmc simplif switch
-   value_rec_compiler
-   translmode
-   transl_comprehension_utils
-   transl_array_comprehension transl_list_comprehension
-   translattribute translclass translcore translmod translobj translprim
-
-   ;; bytecomp/
-   debug_event meta opcodes bytesections dll symtable
-
-   ;; some of COMP
-   pparse main_args compenv compmisc makedepend compile_common
-   ; manual update: mli only files
-   annot value_rec_types asttypes cmo_format outcometree parsetree debug_event
- ))
+  ;; UTILS
+  config
```

Comment:
(I think we could make our lives better by
sorting the items, but that's independent
from what we are doing her.)

Reply:
Yeah indeed